src/app/js/admin/enrichment/EnrichmentForm.js

```diff
@@ -117,6 +141,13 @@ export const EnrichmentFormComponent = ({
         setAdvancedMode(!advancedMode);
     };
 
+    const handleLaunchEnrichment = () => {
+        onLaunchEnrichment({
+            id: initialValues._id,
+            action: initialValues?.status === FINISHED ? 'relaunch' : 'resume',
```

Comment:
I'm not sure if we `resume` the enrichment, or if we just `launch` it, to me, `resume` assume that it was somehow paused. Just wording here, no big deal :)